.goreleaser.yaml

```
@@ -0,0 +1,46 @@
+# yaml-language-server: $schema=https://goreleaser.com/static/schema.json
+# vim: set ts=2 sw=2 tw=0 fo=cnqoj
+
```

I would add the `project_name`

```suggestion
project_name: kubectl-epinio

```